### PR TITLE
1044 add footer for all pages

### LINKED ISSUE #1044: Add footer for all pages
Footer should be displayed for all pages.

Currently footer is displayed only for catalog home page (/catalog). Put footer div for all pages.

### PR DESCRIPTION
**Pull request by minump**
## Description
Footer is added to all pages
Footer div is moved from home page to topnav page. the topnav page is included in all pages.
Some style changes are done.

**Fixes #1044**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/templates/topbottomnav.html [hunk 1]
```diff
@@ -30,4 +30,9 @@ <h1 class="catalog-home">Contributions Catalog</h1>
 
 </div>
 
+<div class="rokwire-footer">
+    <img src= "{{ url_for('static', filename='powered-by-rokwire-no-i-block.png') }}" alt="footer logo" />
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/topbottomnav.html:34]: I think we may have to use all images mentioned in the srcset attribute in the design. The current rendering looks slightly blurred. 
image

File: contributions/catalog/webapps/templates/contribute/talent_details.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 <body onload=start()>
 <div class="header">
-    {% include 'topnav.html' %}
+    {% include 'topbottomnav.html' %}
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/talent_details.html:23]: I would also suggest using a better readable file name like top_bottom_nav.html. Thanks.

File: contributions/catalog/webapps/templates/topbottomnav.html (hunk 1 above)
Comment (minump) [reply contributions/catalog/webapps/templates/topbottomnav.html]: updated

File: contributions/catalog/webapps/templates/contribute/talent_details.html (hunk 2 above)
Comment (minump) [reply contributions/catalog/webapps/templates/contribute/talent_details.html]: updated

**Comment by minump** [comment]
Updated with feedback. Ready for review

File: contributions/catalog/webapps/templates/topbottomnav.html (hunk 1 above)
Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/topbottomnav.html]: Thanks.

File: contributions/catalog/webapps/templates/contribute/talent_details.html (hunk 2 above)
Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/talent_details.html]: Thanks.

**Review by sandeep-ps — DISMISSED** [review]

**Comment by sandeep-ps** [comment]
Thanks for the changes. Approved.

**Comment by minump** [comment]
Implement footer in another page , say footer.html (separate from topnav.html). Include footer.html in all pages, at the bottom of all content.
By this, we can remove the rokwire-footer margin:437px styling, keep the position of the footer below all contents. Adjust the main container margin and padding accordingly.

**Comment by minump** [comment]
Converting to draft to work on changes.

**Comment by minump** [comment]
> Implement footer in another page , say footer.html (separate from topnav.html). Include footer.html in all pages, at the bottom of all content. By this, we can remove the rokwire-footer margin:437px styling, keep the position of the footer below all contents. Adjust the main container margin and padding accordingly.

Added a footer.html page. Removed margin for footer class. Placed footer.html at the bottom of all pages.

**Comment by minump** [comment]
@sandeep-ps Detect-secrets are failing. I cross-checked and dont think I pushed any secrets.
@sandeep-ps and @bingzhang Ready for review.

**Comment by sandeep-ps** [comment]
Hi, Minu, I have restarted reviewing this. I'm getting content blocked by the footer. Should we see this after your new updates? Could you please look into this?

image

**Comment by minump** [comment]
Footer image with padding: 30px 412px 0px 412px;
Screen Shot 2023-02-09 at 3 42 15 PM

Footer image with padding: 85px 363px 26.8px;   (as per zeplin design)
Screen Shot 2023-02-09 at 3 41 48 PM

The footer is bigger as per zeplin design. This is currently implemented in the PR.

**Comment by minump** [comment]
@sandeep-ps updated footer styling. ready for review.

**Comment by minump** [comment]
@sandeep-ps styling and footer updated.. I did check all pages and in incognito.. Hopefully I caught all errors/position misplacements. Let me know if you find something amiss.

@sandeep-ps @bingzhang Ready for review.

**Comment by sandeep-ps** [comment]
The changes look good to me. Thanks.

I saw another thing while reviewing this. Both on edit and create pages, there is some blank space on both the left and right sides. Please create a follow-up issue to fix this, which can be planned for the next sprint. Thanks.

image

**Review by sandeep-ps — APPROVED** [review]

**Review by bingzhang — APPROVED** [review]

**Comment by minump** [comment]
> The changes look good to me. Thanks.
> 
> I saw another thing while reviewing this. Both on edit and create pages, there is some blank space on both the left and right sides. Please create a follow-up issue to fix this, which can be planned for the next sprint. Thanks.
> 
> image

Issue created for further UI change in footer. https://github.com/rokwire/rokwire-building-blocks-api/issues/1081

**Comment by minump** [comment]
Merging to develop. Bypassing brach protections.

**Pull request merged**